docs/best_practices/general.rst

```diff
@@ -41,22 +45,31 @@ Check function: :py:meth:`~nwbinspector.checks.general.check_name_slashes`
 
 
 
-Descriptions
-------------
-
-
-
 .. _best_practice_description:
 
 Metadata and Descriptions
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Names are not for storing meta-data, so meta-data should not be stored solely in the name of objects. It is OK to name
-an object something like “ElectricalSeries_large_array” however the name alone is not sufficient documentation. In this
-case, the source of the signal will be clear from the device of the rows from the linked electrodes table region, and
-you should also include any important distinguishing information in the description field of the object. Make an effort
-to make meta-data as explicit as possible. Good names help users but do not help applications parse your file.
-
-As such, it is not recommended to use blank or default 'placeholder' descriptions.
+The ``name`` of an object is for identifying that object within the file; it is not for storing metadata. Instead, the
+``description`` field for most objects can be used to include any important distinguishing information. When writing
+metadata, make every effort to be as explicit and self-contained as possible in the explaining relevant details. Use
+official ontologies whenever specific references are required, and provide external links to those ontologies.
+
+It is OK to name an object something like ``ElectricalSeriesDuringSomeEvent``, however the name alone is not sufficient
+documentation of the condition ``DuringSomeEvent``. In this case, the source of the signal will be clear from the
+:nwb-schema:ref:`device` linkage in the rows of the passed :nwb-schema:ref:`sec-sec-dynamictableregion` subsetting
+the full :nwb-schema:ref:`ElectrodeTable <groups-general-extracellular-ephys-electrodes>`, so you would not need to
+add any explicit metadata explaining these details. Likewise, the event itself would be specified as an independent
```

Comment:
I'm confused by this. What do the electrodes have to do with the event?

Reply:
The electrodes bit is residual from the original BP:
![image](https://user-images.githubusercontent.com/51133164/160892947-aa4a12c9-3d46-4d83-9bf9-a07e5c6684f9.png)

It's of course independent of an event, so this could be re-arranged a bit